### DISCUSSION TITLE
Why is it missing <groupId>org. apache.maven. plugins</groupId> in the pom

### DISCUSSION THREAD

**Discussion by laglangyue** [netty/netty#13664] [Q&A] [answered]
I find many plugin miss the groupId

**Comment by normanmaurer** [comment]
Just because it also works without... That said feel free to open a PR to fix it.

**Comment by slandelle** [comment, marked as answer]
`org. apache.maven. plugins` is the default value for plugins' `groupId`. IMO, no need for the extra boilerplate.

**Reply by laglangyue** [reply]
Thank you for your clarification，I just learned that it is the default value.

It can indeed be compiled and packaged, but it is not very good for developer on idea, when I use idea on branch 4.1, the plugins will not download.

**Reply by laglangyue** [reply]
![image](https://github.com/netty/netty/assets/35491928/b8976a30-6ed1-4ddf-b61b-c46fbe784656)
![image](https://github.com/netty/netty/assets/35491928/db04f32b-f131-4a31-9967-d9dbd24cfadc)
like this

**Reply by slandelle** [reply]
Then that's a bug to be reported to Idea.
Bugs should be fixed where they happen instead of implementing workarounds downstream.

**Reply by slandelle** [reply]
But make sure to first check that you're using the latest Idea update :)
